WordPressAuthenticator/Authenticator/WordPressAuthenticator.swift

```diff
@@ -177,6 +177,10 @@ import WordPressKit
             return nil
         }
 
+        if let loginNavController = controller as? LoginNavigationController, let loginPrologueViewController = loginNavController.viewControllers.first as? LoginPrologueViewController {
```

Comment:
Just a suggestion, it looks it could be shortened based on previous implementation:

```suggestion
        let loginPrologueViewController = controller.children.first as? LoginPrologueViewController {
```